README.md

```diff
@@ -40,3 +41,47 @@ In your eslint.json file enable this rule with the following:
     }
 }
 ```
+
+## Advanced configuration
+
+```js
+{
+    "plugins": ["no-unsanitized"],
+    "env": {
+        // $ruleName is either "method" or "property"
+        "no-unsanitized/$ruleName": ["errorlevel",
```

Comment:
Yeah this likely should be called schema. I think it's still valuable to have. Other examples to come and also the defaults for jQuery etc